scylla/src/frame/response/result.rs

```diff
@@ -460,3 +517,63 @@ pub fn deserialize(buf: &mut &[u8]) -> StdResult<Result, ParseError> {
         }
     })
 }
+
+#[cfg(test)]
+mod tests {
+    use crate as scylla;
+    use scylla::frame::response::result::CQLValue;
+
+    #[test]
+    fn test_cql_panic() {
+        let mut my_vector: Vec<CQLValue> = Vec::new();
+        my_vector.push(CQLValue::Int(20));
+        let mut nd_vec: Vec<CQLValue> = Vec::new();
+        nd_vec.push(CQLValue::List(my_vector));
+    }
+
+    #[test]
+    fn test_vec_from_cql() {
+        let mut my_vec: Vec<CQLValue> = Vec::new();
+
+        my_vec.push(CQLValue::Int(20));
+        my_vec.push(CQLValue::Int(2));
+        my_vec.push(CQLValue::Int(13));
+
+        let cql: CQLValue = CQLValue::List(my_vec);
+        let decoded = cql.into_vec().unwrap();
+
+        assert_eq!((decoded[0]), CQLValue::Int(20));
```

Comment:
Nit: parentheses around `decoded[0]` are unnecessary. Please remove them here and in other assertions in this file, too.

Reply:
Fixed.